composer.json

```diff
@@ -15,9 +15,9 @@
     "require": {
         "php": "^5.6|^7.0",
         "surfnet/yubikey-api-client": "^2.1",
-        "symfony/config": "^2.7|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/http-kernel": "^2.7|^3.0"
+        "symfony/config": "^2.7|^3.0|^4.4",
```

Comment:
Have you checked if any of our products is still using SF 2.7, I think it's safe to drop SF2 support. But leaving SF3 support for now might be a good idea. But maybe tighten the constraint to 3.4.